cx-sig-charter-draft.md

```diff
@@ -0,0 +1,81 @@
+  - the CDF contribution gamification app [[WIP](https://github.com/ExitoLab/spinnaker_gamification_app)]
+  - contributor experience survey(s)
+  - engagement on project platforms
+  - pull request and issue engagement and closure times
+- Set community health goals such as improved PR and issue engagement and closure times, and track progress towards their achievement using established metrics.
+- Research other OSS projects and consult with their leaders about contributor experience topics to make sure we are always delivering value to our contributors
+- Create and maintain subprojects that incentivize community outreach, individual content creation, and welcoming/onboarding of new contributors
+
+#### Cross-project and Externally Facing Processes
+
+We cross-cut all SIGs and working groups to deliver the following processes:
+
+- Deploying Changes:
+ This SIG will create change-deployment policies that balance quick response to the needs of the community and a limit on disruptions for project contributors. We will create policies to socialize and elicit feedback on changes via community channels such as Slack, spinnaker.io, Twitter, mailing lists, SIG meetings, delegating to SIG leads, etc. The amount of notice we provide and the amount of consensus we seek is driven by our estimation of risk. We can estimate risk based on these parameters:
+
+- Low-risk changes impact a small number (<3) of SIGs, working groups, or repositories, do not break existing contributor workflows, and are easy to roll back.
+- High-risk changes impact a large number (>3) of SIGs, working groups, or repositories, break existing contributor workflows and are not easy to roll back.
+- Depending on how wide of an ecosystem change this is, we may also slack, blog, tweet, and use other channels to get the word out.
+- Our standard time box is 72 business hours; however, there may be situations where we need to act quickly but the time period will always be clear and upfront.
+- Encourage automation to improve productivity for contributors where it makes sense, and consult with SIG leads on automation workflows.
+
+If we need funding for any reason, we approach [CDF](https://cd.foundation).
+CDF may contribute funding to our platforms, processes, and/or programs. They do not play a day-to-day operations role and have bestowed that to our community to run as we see fit. Since they do fund some of our initiatives, this means that they hold owner account privileges on certain platforms like Zoom and Slack. The CX SIG will work to ensure that project maintainers are included as admins.
+
+### Out of scope
+
+- Code for the testing and CI infrastructure
+- User community management.
+- The contributor experience for repositories not included in the Spinnaker organization
+- We do not create SIGs/working group but can assist in the various community management needs of their micro-communities that would kick off their formation and keep them going.
```

Comment:
I'm unclear on how this point co-exists with "user community management" being out of scope.

Reply:
We can help with the formation of SIG's and/or working groups but the idea initiation is up to the users wishing to form so said group. That is different than community management.

Reply:
I see how this wording is unclear and will think about a better suggestion. I think it is trying to say that general user community management is out of scope, as in, this SIG is not wholly or primarily responsible for user community management (which is in scope for the Steering Committee in our docs as written, cc @imosquera). However, this group can assist in ensuring that SIGs to address projects needs are created, and that critical/productive SIGs are maintained as people move in and out of the project. They can do this through enablement and relationship management, which intersects with, and is ideally positioned by, other work described in the charter.